### DISCUSSION TITLE
Change username stuck

### DISCUSSION THREAD

**Discussion by petertwise** [community/community#23809] [New to GitHub]
<p>Hi!</p>
<p>I’m trying to change my username from mybusinessname to myfirstnamelastname so I can use the exiting personal username on an Organziation account.</p>
<p>I started the process, but it seems to be stuck. <a href="https://github.com/settings/admin" rel="nofollow noopener">https://github.com/settings/admin</a> says “Your username is being changed. This may take a few minutes.” - it has said that for about 24h now, which seems like too long for something that should take “a few minutes” to process.</p>
<p>I opened a ticket for this, but it seems like response times are slow due to the pandemic. Can anyone here offer any insight or solutions? Or is this normal and I just have to wait longer?</p>
<p>Thanks!</p>
<ul>
<li>Peter</li>
</ul>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/983026f07075ddb4cf4427479101728b0d0b78f1.png" data-download-href="/uploads/short-url/lIjJW62mTmWJjdLnYXdbuwNuIcV.png?dl=1" title="Screenshot 2020-05-15 08.13.41.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/450137/181079383-247dac9a-5966-4172-93f0-a3d5da44aa7c.png" alt="Screenshot 2020-05-15 08.13.41.png" data-base62-sha1="lIjJW62mTmWJjdLnYXdbuwNuIcV" width="690" height="163" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/983026f07075ddb4cf4427479101728b0d0b78f1_2_690x163.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/983026f07075ddb4cf4427479101728b0d0b78f1_2_1035x244.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/983026f07075ddb4cf4427479101728b0d0b78f1.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/983026f07075ddb4cf4427479101728b0d0b78f1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-05-15 08.13.41.png</span><span class="informations">1128×268 23.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by petertwise** [comment]
<p>Any word on this? I’m a pro user and submitted a support request 8 days ago… no response here or from the support ticket.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for your patience! I see our support team solved the issue.</p>

**Comment by petertwise** [comment, marked as answer]
<p>Yes, they did – thank you!!!</p>
